environment.yml

```diff
@@ -3,305 +3,16 @@ channels:
-  - pyopenssl=21.0.0=pyhd8ed1ab_0
-  - pyparsing=3.0.6=pyhd8ed1ab_0
-  - pyqt=5.12.3=py39hf3d152e_8
-  - pyqt-impl=5.12.3=py39hde8b62d_8
-  - pyqt5-sip=4.19.18=py39he80948d_8
-  - pyqtchart=5.12=py39h0fcd23e_8
-  - pyqtwebengine=5.12.1=py39h0fcd23e_8
-  - pyrsistent=0.18.0=py39h3811e60_0
-  - pysocks=1.7.1=py39hf3d152e_4
-  - python=3.9.7=hb7a2778_3_cpython
-  - python-dateutil=2.8.2=pyhd8ed1ab_0
-  - python-lsp-black=1.0.0=pyhd8ed1ab_0
-  - python-lsp-jsonrpc=1.0.0=pyhd8ed1ab_0
-  - python-lsp-server=1.2.4=pyhd8ed1ab_0
-  - python-slugify=5.0.2=pyhd8ed1ab_0
-  - python_abi=3.9=2_cp39
-  - pytz=2021.3=pyhd8ed1ab_0
-  - pyxdg=0.27=pyhd8ed1ab_0
-  - pyyaml=6.0=py39h3811e60_3
-  - pyzmq=22.3.0=py39h37b5a0c_1
-  - qdarkstyle=3.0.2=pyhd8ed1ab_0
-  - qstylizer=0.2.1=pyhd8ed1ab_0
-  - qt=5.12.9=hda022c4_4
-  - qtawesome=1.1.0=pyhd8ed1ab_0
-  - qtconsole=5.1.1=pyhd3eb1b0_0
-  - qtpy=1.11.2=pyhd8ed1ab_0
-  - re2=2021.11.01=h9c3ff4c_0
-  - readline=8.1=h46c0cb4_0
-  - regex=2021.11.10=py39h3811e60_0
-  - requests=2.26.0=pyhd8ed1ab_0
-  - rope=0.21.1=pyhd8ed1ab_0
-  - rtree=0.9.7=py39hb102c33_3
-  - s2n=1.3.0=h9b69904_0
-  - scipy=1.7.2=py39hee8e79c_0
-  - secretstorage=3.3.1=py39hf3d152e_1
-  - setuptools=59.2.0=py39hf3d152e_0
-  - six=1.16.0=pyh6c4a22f_0
-  - snappy=1.1.8=he1b5a44_3
-  - snowballstemmer=2.2.0=pyhd8ed1ab_0
-  - sortedcontainers=2.4.0=pyhd8ed1ab_0
-  - sphinx=4.3.0=pyh6c4a22f_0
-  - sphinxcontrib-applehelp=1.0.2=py_0
-  - sphinxcontrib-devhelp=1.0.2=py_0
-  - sphinxcontrib-htmlhelp=2.0.0=pyhd8ed1ab_0
-  - sphinxcontrib-jsmath=1.0.1=py_0
-  - sphinxcontrib-qthelp=1.0.3=py_0
-  - sphinxcontrib-serializinghtml=1.1.5=pyhd8ed1ab_1
-  - spyder=5.1.5=py39hf3d152e_1
-  - spyder-kernels=2.1.3=py39hf3d152e_1
-  - sqlite=3.36.0=h9cd32fc_2
-  - tblib=1.7.0=pyhd8ed1ab_0
-  - testpath=0.5.0=pyhd8ed1ab_0
-  - text-unidecode=1.3=py_0
-  - textdistance=4.2.2=pyhd8ed1ab_0
-  - three-merge=0.1.1=pyh9f0ad1d_0
-  - tinycss2=1.1.0=pyhd8ed1ab_0
-  - tk=8.6.11=h27826a3_1
-  - toml=0.10.2=pyhd8ed1ab_0
-  - tomli=1.2.2=pyhd8ed1ab_0
-  - toolz=0.11.2=pyhd8ed1ab_0
-  - tornado=6.1=py39h3811e60_2
-  - traitlets=5.1.1=pyhd8ed1ab_0
-  - typed-ast=1.5.0=py39h3811e60_0
-  - typing_extensions=4.0.0=pyha770c72_0
-  - tzdata=2021e=he74cb21_0
-  - ujson=4.2.0=py39he80948d_1
-  - unidecode=1.3.2=pyhd8ed1ab_0
-  - urllib3=1.26.7=pyhd8ed1ab_0
-  - watchdog=2.1.6=py39hf3d152e_1
-  - wcwidth=0.2.5=pyh9f0ad1d_2
-  - webencodings=0.5.1=py_1
-  - wheel=0.37.0=pyhd8ed1ab_1
-  - wrapt=1.12.1=py39h3811e60_3
-  - wurlitzer=3.0.2=py39hf3d152e_1
-  - xarray=0.20.1=pyhd8ed1ab_0
-  - xorg-libxau=1.0.9=h7f98852_0
-  - xorg-libxdmcp=1.1.3=h7f98852_0
-  - xz=5.2.5=h516909a_1
-  - yaml=0.2.5=h516909a_0
-  - yapf=0.31.0=pyhd8ed1ab_0
-  - zarr=2.10.3=pyhd8ed1ab_0
-  - zeromq=4.3.4=h9c3ff4c_1
-  - zict=2.0.0=py_0
-  - zipp=3.6.0=pyhd8ed1ab_0
-  - zlib=1.2.11=h36c2ea0_1013
-  - zstd=1.5.0=ha95c52a_0
-prefix: /home/kimbo/anaconda3/envs/disdrodb
+  - click
+  - dask
+  - h5py
+  - netCDF4
+  - numpy
```

Comment:
I guess `numpy`, `pandas `and `dask ` could be removed since are already requirements of xarray.

Reply:
Absolutely, the new file has been tested also for the sphiny doc generation. @ghiggi you can validate this pr